packages/web-components/src/components/rux-input/rux-input.scss

```diff
@@ -186,6 +196,18 @@
     }
 }
 
+:host-context(.light-theme) {
+    --calendar-icon: var(--calendar-icon-light);
+    --time-icon: var(--time-icon-light);
+    --search-icon: var(--search-icon-light);
+}
+
+:host-context(.dark-theme) {
```

Comment:
This entire selector for `:host-context(.dark-theme)` can be removed. 

You're already setting the dark icons as defaults up above, and additionally we don't have a concept of a '.dark-theme' class since dark theme is our default for Astro. Either way, you'd want the cascade for an applied `.light-theme` class to override the icon custom properties so it should appear after any default selectors.